css/styles.css

```diff
@@ -0,0 +1,177 @@
+.rules-container {
+    width: 95vw;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.hidden {
+    display: none;
+}
+
+.rules-header {
+    display: flex;
+    justify-content: center;
+    align-self: center;
+}
+
+.rules-title {
+    margin-right: 1rem;
+}
+
+.rules-heading {
+    padding-left: 2.5rem;
+}
+
+ul {
+    font-size: 1.2rem;
+}
+
+* {
+    font-family: 'Bebas Neue', sans-serif;
+    font-weight: 800;
+}
+
+h1 {
+    text-align: center;
+}
+
+hr {
+    width: 80%;
+    border: 0.1rem solid #333333;
+}
+
+.game {
+    width: 90%;
+    margin: 2rem auto;
+    display: flex;
+    flex-direction: column;
+
+}
+
+.game span {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 3rem;
+    height: 3rem;
+    background-color: #CCCCCC;
+    border: 0.15rem solid #333333;
+    border-radius: 0.5rem;
+    gap: 0.2rem;
+    font-size: 1.8rem;
+}
+
+.row0, .row1, .row2,
+.row3, .row4, .row5 {
+    display: flex;
+    justify-content: center;
+    gap: 0.5rem;
+    margin: 0.1rem;
+}
+
+@media only screen and (min-width: 700px) {
+
+    .game span {
+        width: 4rem;
+        height: 4rem;
+    }
+}
+
+.keyboard {
+    width: 100%;
+    position: fixed;
+    bottom: 0;
+}
+
+.container {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background: #FFFFFF;
+}
+
+button {
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+}
+
+div[class*='line'] {
+    display: flex;
+    justify-content: space-around;
+    gap: 0.2rem;
+}
+
+div[class*='line'] + div[class*='line'] {
+    margin-top: 0.5rem;
+    gap: 0.2rem;
+}
+
+.line3 {
+    padding-bottom: 0.5rem;
+}
+
+.line2 {
+    padding: 0 30px;
+}
+
+div[class*='line'] button {
+    color: #000000;
+    padding: 0.8rem;
+    background: #CCCCCC;
+    border: 0.15rem solid #333333;
+    border-radius: 0.5rem;
+    width: 1.5rem;
+    text-align: center;
+}
+
+
+div[class*='line'] button:hover {
+    cursor: pointer
+}
+
+div[class*='line'] button:active {
+    transform: scale(.9);
+}
+
+.line3 button:first-of-type {
+    width: 4.5rem;
+    padding: 0;
+}
+
+.line3 button:last-of-type {
+    width: 2.5rem;
+    padding: 0;
+}
+
+    @media only screen and (min-width: 700px) {
+        div[class*='line'] button {
+            width: 3rem;
+        }
+
+        .line3 button:first-of-type {
+            width: 7rem;
+            padding: 0;
+        }
+
+        .line3 button:last-of-type {
+            width: 5rem;
+            padding: 0;
+        }
+    }
+
+        .correct {
```

Comment:
Check indenting, and combine with the other media query of the same size

Reply:
done